test/support/script.sh

```diff
@@ -0,0 +1,350 @@
+#!/bin/sh
```

Comment:
This is a big no-no. It would be harder to maintain, but I don't see a point either.